addons/mail/sfu_server/README.md

```diff
@@ -0,0 +1,76 @@
+# Odoo: Discuss SFU
+
+## Overview
+
+Contains the code for an SFU (Selective Forwarding Unit) server 
+to be used alongside the Odoo Discuss module. The SFU server is responsible for
+handling the WebRTC connections between users and providing channels to coordinate
+these connections.
+
+## Prerequisites
+
+- An Odoo server with the Discuss module installed
+- [Node.js](https://nodejs.org/)
+
+## Deployment
+
+1. Install dependencies.
+    ```bash
+        npm install
+    ```
+2. Run the build script to generate the client bundle (requires dev dependencies).
```

Comment:
> requires dev dependencies

That is confusing... It basically means the dev dependencies are required, unless we would build the client bundle externally.